src/rectangle.rs

```diff
@@ -1,15 +1,26 @@
-use crate::{Mm, Point};
+//! Utilities for rectangle paths.
 
+use crate::{
+    Mm, Point, OP_PATH_CONST_CLIP_EO, OP_PATH_CONST_CLIP_NZ, OP_PATH_CONST_RECT, OP_PATH_PAINT_END,
+    OP_PATH_PAINT_FILL_NZ, OP_PATH_PAINT_FILL_STROKE_CLOSE_NZ, OP_PATH_PAINT_STROKE,
+};
+
+/// A helper struct to insert rectangular shapes into a PDF.
+///
+/// This can be used to paint rectangles or to clip other paths.
 #[derive(Debug, Copy, Clone)]
 pub struct Rect {
-    /// x position from the bottom left corner in pt
+    /// Position of the lower left point of the rectangle, relative to the bottom left corner.
```

Comment:
I'd add "bottom left corner of the PDF page" and "in point" at the end so it's immediately clear when reading the documentation.

Reply:
Ok, just keep in mind that this is not an invariance provided by `Rect`, but it is related to how `Point` works. If you mean to change how `Point` work then you need to remember to update this documentation.